IEEE-PSH/hackpsh-web
# Added Basic Header Titles to Most Pages

# Description
- Add titles to Home, About, Partners, Challenges, Announcements, Dashboard
- All nav links can be accessed except for Leaderboard (to prevent merge conflicts with 'leaderboard-prototype')
# Test Guidelines
- See if you can visit each page (except Leaderboard) through the navigation menu.
